course6/toyapp_mikhail/displaylib/vector2d.h

```diff
@@ -1,26 +1,9 @@
 #pragma once
 
-namespace Display {
+#include "vector_common.h"
 
-	class Vector2D
-	{
-	public:
-		double x, y;
+namespace Display {
 
-		Vector2D operator + (const Vector2D& v)
-		{
-			auto r = Vector2D(*this);
-			r.x += v.x;
-			r.y += v.y;
-			return r;
-		}
+using Vector2D = VectorCommon<double, 2>;
```

Comment:
I would say it's statistical overkill to have a separate source file for one line

Reply:
Fixed. Left just one file with template definitions